examples/tcp_rerouting/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+publish = false
+name = "proxy-wasm-example-tcp-rerouting"
+version = "0.0.1"
+authors = ["Proxy-Wasm contributors"]
+description = "Proxy-Wasm plugin example: TCP Rerouting based on source IP"
+license = "Apache-2.0"
+edition = "2021"
+rust-version = "1.65"
```

Comment:
Does this work? We had to bump MSRV in the SDK because `cargo` always selected the latest version of `quote` crate, ignoring SDK's `rust-version`.